overlay.d/05rhcos/usr/lib/dracut/modules.d/01scsi-rules/module-setup.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+# -*- mode: shell-script; indent-tabs-mode: nil; sh-basic-offset: 4; -*-
+# ex: ts=8 sw=4 sts=4 et filetype=sh
```

Comment:
Can you add a comment here on why we need this as well as a link to the RHBZ for posterity?